terraform-ecs-fargate-autoscaling-basic/autoscaling.tf

```diff
@@ -20,7 +20,7 @@ resource "aws_appautoscaling_policy" "up" {
   step_scaling_policy_configuration {
     adjustment_type = "ChangeInCapacity"
     cooldown = 60
-    metric_aggregation_type = "Maximum"
+    metric_aggregation_type = var.metric_aggregation_type
```

Comment:
@trubi Tenhle modul taky odpovídá https://github.com/FigurePOS/fgr-service-orders/tree/master/tf/ecs_fargate_autoscaling tak bych ho použil odtud. Akorát se dívám se že všechny služby kromě business configu mají `Maximum`, business config má `Average` (https://github.com/search?q=org%3AFigurePOS%20metric_aggregation_type&type=code). Tak tu hodnotu parametrizuju.

Reply:
Tady ten modul je připravený na to aby se používal odtud centrálně a jen se to nedotáhlo, nebo proč má každá servisa svůj vlastní klon? (Tady ten centrální používá jen `make-it-butter-api`: https://github.com/search?q=org%3AFigurePOS+terraform-ecs-fargate-autoscaling-basic&type=code)

Reply:
btw. tohle asi nejsem jisty, ze to ma nejaky specificky duvod ze je tam rozdil v Maximum a Average.. mozna je to nahoda nebo nejaka zmena co jsme udelali a uz se to nepreneslo vsude jinde? Realne bych rekl, ze average by melo fungovat lip

Reply:
ono jako asi stejne vetsina tech parametru bude dobra mit nekdy casem parametrizovanych... protoze to proste kazda sluza muze chtit jinak.. ted to zatim nebylo potreba, prece jen to nescaluje do nejakych zavratnych vysek zatim

Reply:
Ahá, ok, average je default, jsem právě nevěděl jestli to maximum nemá nějaký specifický důvod. Dám teda Average jako default a nechám to konfigurovatelný.

Reply:
Jo, mozna to melo nejaky duvod, ale fakt si nepamatuju